### DISCUSSION TITLE
UnicodeDecodeError: 'gbk' codec can't decode byte 0xae in position 86: illegal multibyte sequence

### DISCUSSION THREAD

**Discussion by Heacatew** [mikumifa/biliTickerBuy#220] [提问专区]
求助配置完到了开始抢票页面报这个错误怎么解决"UnicodeDecodeError: 'gbk' codec can't decode byte 0xae in position 86: illegal multibyte sequence"

**Comment by Heacatew** [comment]
已解决，怀疑是exe文件所在文件夹下有其他格式的文件导致的，重新创建空文件夹再解压可以正常运行

**Reply by tecsm** [reply]
同样的问题，这个其他文件为啥会影响脚本呢。。

**Reply by tecsm** [reply]
@mikumifa

**Reply by tecsm** [reply]
我的是UnicodeDecodeError: 'gbk' codec can't decode byte 0xae in position 67: illegal multibyte sequence

**Reply by tecsm** [reply]
我刚删除重新解压，随便找了个展，5s成功了。不知道问题在哪= =
